packages/preview-middleware-client/src/adp/quick-actions/table-quick-action-base.ts

```diff
@@ -131,16 +143,61 @@ export abstract class TableQuickActionDefinitionBase {
             }
         }
         if (this.children.length > 0) {
-            this.isActive = true;
+            this.isApplicable = true;
         }
     }
 
+
+    /**
+     * Retrieves the internal table from a UI5Element and checks if it contains rows.
```

Comment:
The function doesn't check anything. It just returns table rows.
I'd suggest to make it a reusable function that returns internal table of the control. 
What to check in this table - it should be business of the function consumer